src/Robot/Testing.hs

```diff
@@ -19,20 +19,124 @@ import Data.List
 import Control.Monad
 import Debug.Trace
 
+-- <<< HELPER FUNCTIONS >>>
+
 -- Create a tableau with just one given target
 exprToTab :: Expr -> Tableau
 exprToTab e = Tableau (Poset [] []) (Box [] [PureTarg e])
 
--- Parse a string as an expression then turn it into a Tableau as above
+-- Parse a string for a single target as an expression then turn it into a Tableau as above
 stringToTab :: String -> Maybe Tableau
 stringToTab s = do
     e <- parseSimple parseExpr s
     return $ exprToTab e
 
 -- <<< MOVE TESTING >>>
 
--- IMPROVEMENT - have fixed the hacky need for negative InternalName's here using HoleExpr's, but this seemed to hit performance a bit
--- How can we improve this, and why does it affect performance?
+{-
```

Comment:
It would be nice to adjust these to use haddock syntax instead, so that the documentation generated by `stack haddock` includes the new text.